development-defaults.ini

```diff
@@ -113,16 +149,8 @@ no_shirt     = 0
 'Squarewave Lanyard'          = 3
 
 
-
 [enums]
 
-[[badge_status]]
```

Comment:
Yup, that's also an AnthroCon request.

I don't know if you caught it, but I actually already disabled deleting an attendee in ac-merge. The functionality isn't complete, though - it currently does nothing except log the attempt. I've been meaning to fix it up.